client/layout/sidebar/skip-navigation.jsx

```diff
@@ -0,0 +1,29 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import PropTypes from 'prop-types';
+import { localize } from 'i18n-calypso';
+import React from 'react';
+
+class SkipNavigation extends React.Component {
+	static propTypes = {
+		skipToElementId: PropTypes.string,
+	};
+
+	render() {
+		return (
+			<a
+				href={ '#' + this.props.skipToElementId }
+				className="sidebar__skip-navigation"
+				data-tip-target={ this.props.tipTarget }
```

Comment:
I don't think we need the `data-tip-target` here, this is used for guided tour steps and I don't think we'd have a step identifying this link. The `tipTarget` prop is also never used/defined.

Reply:
Thank you for spotting that :)